src/models/create_token_transaction.ts

```diff
@@ -55,7 +55,7 @@ class CreateTokenTransaction extends Transaction {
 
   /**
    * Serialize funds fields
-   * version, len inputs, len outputs, inputs, outputs and token info
+   * signal bits, version, len inputs, len outputs, inputs, outputs and token info
```

Comment:
Comment unrelated with the line. I think you should add the `signalBits` field to this class's `optionsType` and `defaultOptions`. The `version` field does not appear there because a transaction that creates tokens has a fixed version value but the signal bits might change.

Reply:
Done!